openstack/neutron-hypervisor-agents/templates/daemonset-openvswitch-agent.yaml

```diff
@@ -0,0 +1,156 @@
+{{- $hypervisor := merge .Values.defaults.hypervisor.kvm .Values.defaults.hypervisor.common }}
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: neutron-openvswitch-agent
+  labels:
+    system: openstack
+    type: backend
+    component: neutron
+spec:
+  revisionHistoryLimit: {{ .Values.pod.lifecycle.upgrades.deployments.revision_history }}
+  selector:
+    matchLabels:
+      name: neutron-openvswitch-agent
+  template:
+    metadata:
+      labels:
+        {{- tuple . "neutron" "openvswitch-agent" | include "helm-toolkit.snippets.kubernetes_metadata_labels" | nindent 8 }}
+        name: neutron-openvswitch-agent
+        alert-tier: os
+        alert-service: neutron
+        hypervisor: "kvm"
+      annotations:
+        configmap-etc-hash: {{ include (print $.Template.BasePath "/configmap-etc.yaml") . | sha256sum }}
+    spec:
+      hostNetwork: true
+      hostPID: true
+      hostIPC: true
+      hostAliases:
+      {{- range $ip := .Values.rabbitmq.externalIPs }}
+      - ip: {{ $ip }}
+        hostnames:
+        - {{ default "rabbitmq" $.Values.rabbitmq.nameOverride | printf "neutron-%s" | trunc 63 | replace "_" "-" | trimSuffix "-" }}
+      {{- end }}
+      tolerations:
+      {{- range $k, $v :=  merge $hypervisor.pod.tolerations .Values.pod.tolerations }}
+      - # {{ $k }}
+      {{- toYaml $v | nindent 8 }}
+      {{- end }}
+      nodeSelector:
+      {{- range $k, $v := $hypervisor.pod.nodeSelector }}
+        {{ $k }}: {{ $v }}
+      {{- end }}
+      hostAliases:
+      {{- range $ip := .Values.rabbitmq.externalIPs }}
+      - ip: {{ $ip }}
+        hostnames:
+        - {{ default "rabbitmq" $.Values.rabbitmq.nameOverride | printf "neutron-%s" | trunc 63 | replace "_" "-" | trimSuffix "-" }}
+      {{- end }}
```

Comment:
`hostAliases` is already defined in line 29

Reply:
Removed this in favor for the one in line 29